plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -604,6 +608,13 @@ public KubernetesClusterResponse createKubernetesClusterResponse(long kubernetes
         response.setEndpoint(kubernetesCluster.getEndpoint());
         response.setNetworkId(ntwk.getUuid());
         response.setAssociatedNetworkName(ntwk.getName());
+        if (ntwk.getGuestType() == Network.GuestType.Isolated) {
```

Comment:
so when SHARED/L2/VPC this needs changing again?

Reply:
There shouldn't be any need. It doesn't support L2 / Shared (since they don't have public IPs) or VPCs as of now, and if support is added, VPC networks are again isolated networks